validation-test/Reflection/reflect_Enum_values_resilient.swift

```diff
@@ -0,0 +1,74 @@
+// RUN: %empty-directory(%t)
```

Comment:
New test case builds a separate resilient library and verifies that we can project values from enums defined in that library.